script.js

```diff
@@ -28,19 +28,70 @@ randomQuote();
 // Part 1
 
 
+ var mainTitle = document.querySelector('#main-title').textContent.replace('Hi! My name is DOM Toretto.');
+
+
 // Part 2
+document.body.style.backgroundColor = '#af4b64';
 
 
 // Part 3
+var fontSize = document.querySelectorAll('.special-title')
+for (var i=0 ; i < fontSize.length; i++)
+fontSize[i].style.fontSize = '2rem';
 
 
 // Part 4
 
+var addDiv = document.querySelectorAll('.side-bar')[1];
+var addUl = document.querySelectorAll('ul')[1];
+var newLi = document.createElement('li');
+var newLiCity = document.createTextNode('Hial');
+
+newLi.appendChild(newLiCity);
+addUl.appendChild(newLi);
+addDiv.appendChild(addUl);
+this.document.body.appendChild(addUl);
+
+// // Part 5
+
+
+var newDiv = document.createElement('div');
+newDiv.classList.add('blog-post');
+newDiv.classList.add('purple');
+
+console.log(newDiv);
 
-// Part 5
+var newH1 = document.createElement('h1');
+var pText = document.createElement('p');
+var cityNew = document.createTextNode('end game');
 
+pText.appendChild(cityNew);
+newH1.appendChild(newLiCity);
+newDiv.appendChild(newH1);
+newDiv.appendChild(pText);
+
+document.querySelector('.main').appendChild(newDiv);
+
+
+// // Part 6
+
+// var favoriteThings = document.querySelector('#favorite-things');
```

Comment:
You are going in the right direction here. When you state 
```js
var favoriteThings =  document.querySelector('#favorite-things');
```
you are overwriting the favoriteThings array above. using a different name for the dom selection may help you get there!